stake-o-matic.sh

```diff
@@ -15,7 +54,14 @@ if [[ $BUILDKITE = true ]]; then
   git config --global user.name "Solana Maintainers"
 fi
 
-./solana-stake-o-matic "$@"
+if [[ $CLUSTER == "testnet" ]]; then
+  ./solana-stake-o-matic "${TESTNET_ARGS[@]}"
+elif [[ $CLUSTER == "mainnet beta" ]]; then
```

Comment:
IMO kabab this, so mainnet-beta.  We never use "mainnet beta" in the code anywhere